AAEmu.Game/Models/Game/Char/Character.cs

```diff
@@ -138,6 +138,12 @@ public partial class Character : Unit
         public Point LocalPingPosition { get; set; } // added as a GM command helper
         private ConcurrentDictionary<uint, DateTime> _hostilePlayers { get; set; }
 
+        [UnitAttribute(UnitAttribute.LungCapacity)]
+        public uint LungCapacity
+        {
+            get => (uint)CalculateWithBonuses(60000, UnitAttribute.LungCapacity);
+        }
+
```

Comment:
There is a #region for UnitAttributes only a few lines down! However, considering this won't need to be changed often, it would be preferable to put it at the bottom of the attributes region.